packages/core/src/components/StarRating.tsx

```diff
@@ -67,7 +72,9 @@ const StarRating: React.FC<Props> = ({
             }
             size={starSize}
             color={
-              ratingRounded > i ? theme.colors.primary : theme.colors.divider
+              ratingRounded > i
+                ? activeColor || theme.colors.primary
+                : inactiveColor || theme.colors.divider
```

Comment:
I think rather than falling back to these internally I would make them the default value
```suggestion
              ratingRounded > i ? activeColor : inactiveColor
```